app/scripts/controllers/visualizerModal.js

```diff
@@ -136,11 +130,6 @@ angular.module('openhimConsoleApp')
         $scope.ngError.hasNoComponents = true;
       }
 
-      if(settings.mediators === undefined || settings.mediators.length === 0){
-        $scope.ngError.hasErrors = true;
-        $scope.ngError.hasNoMediators = true;
-      }
-
       if(settings.channels === undefined || settings.channels.length === 0){
```

Comment:
Could the settings parameter of this function be renamed visualizer or vis. Setting doesn't seems to make sense here.